iOS/Features/HomeMapScene/Sources/HomeMapScene/HomeMapButtonView.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  HomeMapButtonView.swift
+//  Test
```

Comment:
```suggestion
//  HomeMap
```

파일 헤더 수정 부탁드려용

저랑 민건님은 Scene을 붙이지 않았는데 이것도 한 번 정해야겠네요
Scene을 붙이는 쪽이 더 직관적인 것 같기도 합니다.